.github/workflows/lint-helm-charts.yml

```diff
@@ -7,27 +7,45 @@ on:
     paths:
       - '.github/workflows/lint-helm-charts.yml'
       - 'charts/**'
+  push:
+    branches: [develop]
+    paths:
+      - 'charts/**'
+
+permissions:
+  contents: read
+
+concurrency:
+  group: charts-lint-${{ github.ref }}
+  cancel-in-progress: true
+  
 jobs:
   lint-test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+
       - name: Set up Helm
-        uses: azure/setup-helm@v4.2.0
+        uses: azure/setup-helm@v4.3.1
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.7.0
+
       - name: Ensure documentation is updated
         uses: docker://jnorwood/helm-docs:v1.14.2
-      - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.6.1
+
       - name: Run chart-testing (list-changed)
         id: list-changed
         run: |
           changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
           if [[ -n "$changed" ]]; then
             echo "changed=true" >> "$GITHUB_OUTPUT"
+            echo "$changed"
           fi
+
       - name: Run chart-testing
         if: steps.list-changed.outputs.changed == 'true'
-        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --validate-maintainers=false
+        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --validate-maintainers=false --check-version-increment=true
```

Comment:
`check-version-increment` is already set to true by default, see https://github.com/helm/chart-testing/blob/main/ct/cmd/lint.go#L67